### ISSUE TITLE
Design and Implement Initial Archive Browser Beta

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/planetary-data-cloud#61]
## 💡 Description

See sub-tasks for initial design and implementation tasks

**Comment by jordanpadams** [comment]
📆 January status: Ops concept and design in work. On schedule

**Comment by jordanpadams** [comment]
📆 February status: Deferred from plans after more discussions of architecture. Delivery Manager prioritized ahead of this component.

**Comment by jordanpadams** [comment]
📆 October Status: Task underway. Pursuing using stylesheets + cloudfront approach for abstracting S3 buckets.

**Comment by tloubrieu-jpl** [comment]
@eddiesarevalo , I was able to set up a cloudfront distribution with origin the bucket `pds-messenger-data-copy`. The distribution provides:
1.  an http link to all objects in the s3 bucket, for example https://d1qy67vkpvel43.cloudfront.net/MESSDEM_1001/AAREADME.TXT
2.  a list of all the objects in the bucket but since all the objects of the bucket are on a single page, I think the number of objects is linited to 1000 (to be confirmed). see https://d1qy67vkpvel43.cloudfront.net/

To have a list of objects by directory, if needed by the Aria application, I see 2 options:
- generate offline index.html pages object containing the list of objects in the directory
- develop a lambda@edge function which writes a response containing the list of objects in the the directory. I am blocked on that with the permission creation for the lambda function (need support from MCP).

Note that S3 does not have the concept of directory, I believe, a directory has to be considered as prefix shared in the key of multiple objects. That might be why it is not natural to get the feature from the cloudfront-s3 connection.

We can also ask for help to the SAs on the general need. I will not work more on that this week.

**Comment by eddiesarevalo** [comment]
Thanks @tloubrieu-jpl !

The list of objects looks similar to what aria https://aria-share.jpl.nasa.gov/ has at https://d1qy67vkpvel43.cloudfront.net/

They then drill down the directories using some delimeter logic for example: 
https://d1z62tir4fw0q0.cloudfront.net/?delimiter=/&prefix=20140512-golden_guardian%2F

I'm trying to see if I can run the code.

**Comment by tloubrieu-jpl** [comment]
Status:
- Eddie is testing Aria-Share from what Susan provided
- We need to setup Cognito to test with the javascript client option.

**Comment by jordanpadams** [comment]
📆 November status: Work underway. On schedule

**Comment by jordanpadams** [comment]
Status: We tracked down the codebase, and @eddiesarevalo was able to fix some of the bugs in the code and tweak the interface/layouts.

**Comment by jordanpadams** [comment]
Closing as completed for now. Will need to work out deployment at release time.
